src/frontend/screens/ObservationDetails/TextArea.tsx

```diff
@@ -16,6 +15,7 @@ const TextArea = ({ value, field, onChange }: QuestionProps) => (
       multiline
       scrollEnabled={false}
       textContentType="none"
+      autoFocus={autoFocus}
```

Comment:
I don't see where this is being set to `true`? Looks like it defaults to `false`